.github/workflows/cmake.yml

```diff
@@ -225,6 +230,19 @@ jobs:
       working-directory: ${{github.workspace}}/build
       run: env UR_CTS_ADAPTER_PLATFORM="${{matrix.adapter.platform}}" ctest -C ${{matrix.build_type}} --output-on-failure -L "conformance" --timeout 180
 
+    - name: Quick Coverage Info
+      if: matrix.compiler.c == 'gcc'
+      working-directory: ${{github.workspace}}/build
+      run: ctest -T Coverage
+
+    - name: Upload coverage to Codecov
+      if: matrix.compiler.c == 'gcc'
+      continue-on-error: true
+      uses: codecov/codecov-action@eaaf4bedf32dbdc6b720b63067d99c4d77d6047d # v3.1.4
+      with:
```

Comment:
we could add `flags: adapters` to distinguish this from other coverage report(s)